pkg/install/genevalogging.go

```diff
@@ -0,0 +1,37 @@
+package install
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"reflect"
+	"strings"
+
+	"github.com/openshift/installer/pkg/asset/installconfig"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/Azure/ARO-RP/pkg/genevalogging"
+	"github.com/Azure/ARO-RP/pkg/util/restconfig"
+)
+
+func (i *Installer) ensureGenevaLogging(ctx context.Context) error {
+	g, err := i.loadGraph(ctx)
+	if err != nil {
+		return err
+	}
+	restConfig, err := restconfig.RestConfig(ctx, i.env, i.doc.OpenShiftCluster)
+	if err != nil {
+		return err
+	}
+
+	cli, err := kubernetes.NewForConfig(restConfig)
+	if err != nil {
+		return err
+	}
```

Comment:
After rebasing on master, you can remove these lines and replace occurrences of `cli` with `i.kubernetescli`.